modules/monitoring-configuring-persistent-storage.adoc

```diff
@@ -3,10 +3,15 @@
 // * observability/monitoring/configuring-the-monitoring-stack.adoc
 
 :_mod-docs-content-type: CONCEPT
-[id="configuring_persistent_storage_{context}"]
+[id="configuring-persistent-storage_{context}"]
 = Configuring persistent storage
 
-Running cluster monitoring with persistent storage means that your metrics are stored to a persistent volume (PV) and can survive a pod being restarted or recreated. This is ideal if you require your metrics or alerting data to be guarded from data loss. For production environments, it is highly recommended to configure persistent storage. Because of the high IO demands, it is advantageous to use local storage.
+Run cluster monitoring with persistent storage to gain the following benefits:
+
+* Protect your metrics and alerting data from data loss by storing them to a persistent volume (PV). As a result, they can survive pods being restarted or recreated.
```

Comment:
```suggestion
* Protect your metrics and alerting data from data loss by storing them to a persistent volume (PV). 
```

Reply:
Not really sure if the last sentence is needed? If you feel it is, may need to be re-worded.

Reply:
I'd leave that to https://issues.redhat.com/browse/OBSDOCS-547 (where we're reshaping that page),but sure we can remove that as users should be more familiar with PV, and we have a dedicated page for that now.